### PR TITLE
Move old SDK/Monsteer spike report to brain

### PR DESCRIPTION
**Pull request by eile**
See BlueBrain/Brion#63

### CODE REVIEW

**Comment by rdumusc** [comment]
tests/spikeReportReaderWriter.cpp should be moved as well, good otherwise

**Comment by eile** [comment]
No, the test needs the monsteer brion plugin. I considered it too. ;)

**Comment by rdumusc** [comment]
I am looking at it for the second time and I don't see where the plugin is needed. If one of the individual tests needs it it then it should be moved to spikeStreamReport.cpp. It is not right that the classes moved to brion rely on unit tests from a different project.

**Comment by eile** [comment]
Ok, will look into it.

**Comment by rdumusc** [comment]
+1 now that the unit test has been moved along with the classes. I thought that changelog entries + version bumps were the norm for such changes, both here and in Brion, but up to you.

**Comment by eile** [comment]
> changelog entries + version bumps 

I don't know. On one hand, I would not consider this a stable API yet, so don't draw too much attention to it. On the other hand, it changes the API (namespace only here). Definitely need to bump version.

What is your opinion?

@bilgili: You were working on this class? Shall I wait with this merge?

**Comment by rdumusc** [comment]
+1 for version bump to Monsteer 0.5.0 as many classes / functions were removed from the Project.

I don't see a changelog entry as a way of drawing attention, but simply documenting the list of non-trivial changes so that people have an easy way to see what happened between 2 releases.

Were you talking about the API about the ABI? Technically ABI has changed as well but this is not so important. I am not sure what rule we decided to follow for unstable projects (version_major == 0), but most of the time it follows version_minor because it is unstable and changes constantly. We could also decide to keep version ABI = 1 constant for those projects.

File: monsteer/python/CMakeLists.txt [hunk 1]
```diff
@@ -11,7 +11,26 @@ if( NOT Boost_PYTHON${USE_BOOST_PYTHON_VERSION}_LIBRARY OR
...
+  ${Boost_PYTHON${USE_BOOST_PYTHON_VERSION}_LIBRARY})
+
+set_target_properties(monsteer_python PROPERTIES
+  OUTPUT_NAME "_monsteer" PREFIX ""
+  LIBRARY_OUTPUT_DIRECTORY ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/monsteer)
+
+set_property(GLOBAL APPEND PROPERTY MONSTEER_ALL_DEP_TARGETS monsteer_python)
+
+install(TARGETS monsteer_python
+  LIBRARY DESTINATION ${PYTHON_LIBRARY_SUFFIX}/monsteer)
```
Comment (hernando) [inline monsteer/python/CMakeLists.txt]: This block whole block is not needed, is it?

**Pull request merged**